frontend/src/InlineCard.tsx

```diff
@@ -0,0 +1,56 @@
+import * as React from 'react';
+import {AppStateConsumer} from './AppStateProvider';
+import {unicodeToCard, cardToUnicodeSuit, SuitCard} from './util/cardHelpers';
+import ElementWithProps from './ElementWithProps';
+
+const ColoredDiv = (color: string) =>
+  ElementWithProps('span', {
+    style: {color, paddingLeft: '0.1em', paddingRight: '0.1em'},
+  });
+const Black = ColoredDiv('#000000');
```

Comment:
I have a preference for using the CSS classes (just the unicode suit names) for this because it makes it easier to user CSS -- thoughts?